tests/metrics/network/network-metrics.sh

```diff
@@ -58,40 +64,33 @@ function jitter {
         rm -f $jitter_result
 }
 
+# Test TCP docker->docker bandwidth with multiple network connections (parallelism)
 function bandwidth_multiple_tcp_connections {
         setup
         number_tcp_connections=8
         multiple_tcp_result=$(mktemp)
         $DOCKER_EXE run -d --name=iperf-server ${image} bash -c "iperf -p ${port} -s" > /dev/null && \
 	server_address=`$DOCKER_EXE inspect --format "{{.NetworkSettings.IPAddress}}" $($DOCKER_EXE ps -ql)` && \
-        $DOCKER_EXE run -ti --rm --name=iperf-client ${image} bash -c "iperf -c ${server_address} -P ${multiple_tcp_result} -t ${time}"  > "$multiple_tcp_result"
+        $DOCKER_EXE run -ti --rm --name=iperf-client ${image} bash -c "iperf -c ${server_address} -P ${number_tcp_connections} -t ${time}"  > "$multiple_tcp_result"
```

Comment:
Maybe not strictly part of this PR, but the indentation in this function looks inconsistent.